aksel.nav.no/website/app/_ui/utils/format-text.ts

```diff
@@ -28,4 +48,4 @@ function removeEmojiesFromText(inputString: string) {
     .trim();
 }
 
-export { abbrName, capitalizeText, removeEmojiesFromText };
+export { abbrName, capitalizeText, removeEmojiesFromText, humanizeText };
```

Comment:
Not used anywhere else, so can remove from export
```suggestion
export { abbrName, capitalizeText, removeEmojiesFromText };
```